local/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
We are already ignoring the contents here `https://github.com/the-skycoin-project/skywire/blob/b158522464cfabdf176898b8454628c5b5a8a34d/.gitignore#L16` and I  also do not see the reason why we need to push an empty folder to the repo. Since after starting the visor with sudo we need to delete the folder again anyway to start a visor without sudo.

Reply:
This was something that @jdknives mentioned doing last we spoke. To address the permissions errors between runs as root and not. The issue still remains, with all the files in the folder though not the folder itself. so perhaps that is why it wasn't done. This is an issue I am attempting to address more thoroughly. Doesn't work as intended - will remove.

Reply:
removed